tests/bdd/steps/given/index.js

```diff
@@ -0,0 +1,3 @@
+export * from './openUrl'
```

Comment:
Just putting this here, cause I don't know where else to put it
* We need to put our steps into a different folder location
* Technically we don't own the `/tests` folder the user does
  * We are expecting them to mount a folder over the tests folder
  * So any code there will no longer be accessable
  * So we should move these steps, as well as the other code so it does not get overwritten
  * I'm referring to the things like the `/bbd/support` folder

Reply:
As discussed, this will be handled on a separate ticket I'm already working on [ZEN-534](https://jira.simpleviewtools.com/browse/zen-534)